src/common/InfoDashboard/index.vue

```diff
@@ -2,35 +2,37 @@
   <div class="info-dashboard" :class="{ 'info-dashboard--loading': isLoading }">
     <transition name="fade" mode="out-in">
       <div v-if="web3ProvidersStore.isConnected" class="info-dashboard__wrp">
-        <div class="info-dashboard__header">
-          <div>
-            <div class="info-dashboard__header-title-wrp">
-              <h5 class="info-dashboard__header-title">
-                {{ $t('info-dashboard.header-title') }}
-              </h5>
-              <app-icon
-                v-tooltip="$t('info-dashboard.header-note')"
-                class="info-dashboard__header-title-icon"
-                :name="$icons.exclamationCircle"
-              />
+        <template v-if="isChartShown">
+          <div class="info-dashboard__header">
```

Comment:
According to semantic html it should be `<header>` tag.

Reply:
done